packages/grafana-runtime/src/config.ts

```diff
@@ -70,6 +70,8 @@ export class GrafanaBootConfig implements GrafanaConfig {
   marketplaceUrl?: string;
   expressionsEnabled = false;
   customTheme?: any;
+  awsAllowedAuthProviders: string[] = [];
+  awsAssumeRoleEnabled = false;
```

Comment:
Are these two used at all? Don't see any usage of them in this PR, even though they're introduced here?